contracts/JBProjects.sol

```diff
@@ -107,31 +103,35 @@ contract JBProjects is JBOperatable, ERC721Votes, Ownable, IJBProjects {
             metadataContentOf[projectId][_metadata.domain] = _metadata.content;
         }
 
-        emit Create(projectId, _owner, _metadata, msg.sender);
+        emit Create(projectId, _owner, _metadata, _msgSender());
```

Comment:
where does _msgSender() come from here? ERC721Votes?

Reply:
ERC721Votes and Ownable, all OpenZeppelin contracts use _msgSender() over msg.sender so you can override them and implement metatx logic if you want to.